.circleci/config.yml

```diff
@@ -17,11 +20,26 @@ jobs:
       - run:
           name: Test with Bats
           command: bats tests
+  ruby-build:
+    docker: *docker
+    steps:
+      - checkout
       - run:
           name: Install and uninstall all suites
           command: |
-            ./rubian install stable unstable legacy
+            bash rubian install stable unstable legacy
             for prefix in /opt/rubies/*; do
               "$prefix"/bin/ruby -v
             done
             ruby -v
+workflows:
+  version: 2
+  test-and-ruby-build:
+    jobs:
+      - tests
+      - ruby-build:
+          requires:
+            - tests
+          filters:
+            branches:
+              only: master
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline